HACKING.md

```diff
@@ -350,7 +350,9 @@ unrelated to the commit message.
 Cockpit is a designed project. Anything that the user will see should have
 design done first. This is done on the wiki and mailing list.
 
-Bigger changes need to be discussed on the #cockpit IRC channel or our mailing list
+Bigger changes need to be discussed on the
+[#cockpit:fedoraproject.org](https://matrix.to/#/#cockpit:fedoraproject.org)
```

Comment:
Is that really what we should use? Not https://chat.fedoraproject.org/#/room/#cockpit:fedoraproject.org ?

Reply:
As far as I can tell, this takes you to a login page which only works for people who are fedoraproject members.

The other one gives you a choice of which client and/or service you'd like to use.

Reply:
But there's an adjacent grammar error I should fix :wink:

Reply:
> But there's an adjacent grammar error I should fix wink

Ah no.  The line after it makes it correct.

Reply:
I just tested this with my gh account, and it works. With "my" link you are kind of forced to have a Fedora account. So that seems better.